roles/openstackclient/templates/docker-compose.yml.j2

```diff
@@ -7,6 +7,8 @@ services:
     restart: unless-stopped
     image: "{{ openstackclient_image }}"
     command: sleep infinity
+    env_file:
+      - "{{ openstackclient_configuration_directory }}/openstackclient.env"
```

Comment:
Use environment instead of env_file or add a task to copy the openstackclient.env.j2 file. Would prefer to use ``environment``.